normandy/recipes/tests/test_serializers.py

```diff
@@ -15,6 +15,7 @@ def test_it_works(self, rf):
         assert serializer.data == {
             'name': recipe.name,
             'id': recipe.id,
+            'last_updated': recipe.last_updated.replace(tzinfo=None).isoformat() + 'Z',
```

Comment:
I wish there was a way to assert it was a date in the right format, without getting all ugly. I'm fine with `Whatever()` though.